### PR TITLE
Update code link colors

### LINKED ISSUE #170: Change code link colors?
I noticed that inline code formatted text looks the same whether it's a link or just regular text. For example, these will have the same styling in `readthedocs` - 

- [`SingleR`](https://bioconductor.org/packages/release/bioc/html/SingleR.html)
- `SingleR`

Maybe we need to incorporate some different styling so in-line code formatted text that is a link clearly appears as a link?

### PR DESCRIPTION
**Pull request by sjspielman**
Closes #170

This issue updates code link colors as suggested by @dvenprasad: 

>  The link color should be blue (#003595). 

Once approved to go into `development`, we can cherry-pick into the release PRs, so it can go out as part of the `AnnData` release.

### CODE REVIEW

**Comment by sjspielman** [comment]
Noting the build in this PR looks as expected, e.g. https://scpca--263.org.readthedocs.build/en/263/processing_information.html#mapping-and-quantification-using-alevin-fry

**Review by dvenprasad — COMMENTED** [review]
I changed the color cause I misunderstood the color question. It should be whatever the default link color is. Also added a class to define a hover color on the linked code block (which is a helpful indicator to users to say, hey you can do something here)

File: docs/_static/style.css [hunk 1]
```diff
@@ -36,6 +36,11 @@ a, a:visited {
   color: #145ECC;
 }
 
+a code span
+{
+  color: #003595;
+}
+
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/_static/style.css:43]: let's let it build and see!

[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/_static/style.css:43]: The hover works, but in the case of multiple words in the same code spot, only one word is highlighted at a time. I don't think this is avoidable though given how the docs actually get built up:

```
<a class="reference external" href="[https://salmon.readthedocs.io/en/latest/alevin.html](view-source:https://salmon.readthedocs.io/en/latest/alevin.html)"><code class="docutils literal notranslate"><span class="pre">salmon</span> <span class="pre">alevin</span></code></a>
```

[the remark this answers is not in the document]
Comment (dvenprasad) [reply docs/_static/style.css:43]: It also messed up the hover on the regular link. I think we should nix this for now

[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/_static/style.css:43]: >  I think we should nix this for now

Do you mean nix the hover, or the color change altogether?

[the remark this answers is not in the document]
Comment (dvenprasad) [reply docs/_static/style.css:43]: the hover

[the remark this answers is not in the document]
Comment (dvenprasad) [reply docs/_static/style.css:43]: Maybe we give sometime and see if Nozomi has any suggestions

[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/_static/style.css:43]: @nozomione, wanted to check if you have any thoughts here about the hover text? Do you think there might be a way to improve the hover highlighting (https://github.com/AlexsLemonade/scpca-docs/pull/263#discussion_r1476423528)? Thanks!!

[the remark this answers is not in the document]
Comment (nozomione) [reply docs/_static/style.css:43]: @dvenprasad & @sjspielman, sorry I just saw this comment!

To fix this issue, try this:
```css
a:hover code span {
  color: #3091D1;
}
```
Hope this helps!

[the remark this answers is not in the document]
Comment (sjspielman) [reply docs/_static/style.css:43]: Thanks, I'll give it try!

**Comment by sjspielman** [comment]
Woohoo it works!

**Review by dvenprasad — APPROVED** [review]
WOO!!

🐸 🐸 🐸

**Comment by sjspielman** [comment]
Actually, how might we feel about sending this right into main after merging into dev? CC @allyhawkins

**Comment by allyhawkins** [comment]
> Actually, how might we feel about sending this right into main after merging into dev? CC @allyhawkins

That would be fine with me. The only thing is that the stable version of the docs won't quite match up with the latest release, but we are going to be doing a new release with the h5 change shortly, so it should be fine.

**Comment by sjspielman** [comment]
> but we are going to be doing a new release with the h5 change shortly, so it should be fine.

@allyhawkins Ah good point! As long as this goes in soon-ish it's fine with me, so we'll just make it part of the h5 release.

**Pull request merged**